### ISSUE TITLE
Youtube API quota

### ISSUE THREAD

**Issue by PyvesB** [badges/shields#10570]
We received a mail mid-August that I had missed:
> To verify that all use of the API is in compliance with the [Terms of Service](https://notifications.google.com/g/p/ANiao5rxRQhHZ10ZPVJ3mLJ5opdTvCSTgJfPRXF_tGClDIC-AopZGhY85X6FZJtv6guNMuvDwb3xqIXHo_PLU_RXbbgzGzFg-iBb-WdB8F0mzyCGnX734nZgwYvN0r2HwSQIbS2EP9TY5XtFibcj64VE544P9YpfPa3t5DW8CAQucgB-Qe3JlPwsvWS6Uw8orku9PIEeUcoh2nhAHz6SG-0xyhdUdAcDfQv_I9f1QUBrjtLF57M8KY4jnmPnioEC3zUVNSblyaI) (including the [Developer Policies](https://notifications.google.com/g/p/ANiao5poNWFRRGoIOGI1iG9bhYsJF_wPT2KslWvF7XorEPqBmZUAtFYAq5yFGZtQZIzRlgKEDZHz71cmTW_2m8Wtnf5EAJc2gx7QcvdED4HD984sj0KRwYLR3G38siev0UgsJhgpnk6KOu1o9x5wv9yPBEz0Rps56EiuMoV-QqLXcyIS8UphDA8kQE_34KWf1u1aq34Id-icZ88z2qSY6LXZY2GuEpUX-IXa_aOQ15GLP7MM0BQk4YoWPlNZ)), we are conducting a periodic audit of API developers. Please complete the audit form, linked below within the next 30 days to ensure continued access to the service. For the reason for filling this form, please choose "I have been selected for a Periodic Audit".

And a couple of weeks ago:
> Because we have not yet received your completed form, the API quota allocated to your projects has been adjusted as follows:
--
Project shields-io-XXXX from 200,000 to 66,600 units
Please complete the form as soon as possible. Your original quota will be restored after successful completion of the audit.

We are currently performing between 70k and 80k requests on weekdays, yet we've not yet been rate limited despite the lower quota. However, to be on the safe side as there's little indication on whether they'll further reduce our limit, I'll submit their form a little later. The last audit was barely over a year ago and was a flamboyant display of Google's bureaucracy and appalling developer experience, I'm hoping this will go more smoothly this time around.

**Comment by PyvesB** [comment]
YouTube is now asking for things like the physical address of our organisation. I had to put a bogus value, hopefully won't cause any trouble.

**Comment by chris48s** [comment]
After we got the most recent rate limit increase, we decreased the time we cache youtube badges for. Last time we had issues with this, we set them to cache for 2 hours which massively reduced our usage.

We could always do this again
https://github.com/badges/shields/pull/9372

For a service like youtube where we are presenting number of views/subscribers/comments/likes etc which are indicative stats (this isn't build status passing/failing) and usually presented as rounded numbers we can get away with cacheing them for longer than we do at the moment without much impact.
